spaceship/lib/spaceship/tunes/app_version.rb

```diff
@@ -609,6 +626,34 @@ def upload_trailer!(trailer_path, sort_order, language, device, timestamp = "05.
         setup_trailers
       end
 
+      # Uploads, app review attachments
+      #
+      # while submitting for review, ITC allow developers to attach file.
+      #
+      # Following list can be found at https://appstoreconnect.apple.com
+      # on iOS app edit version, above the attachment label/button there is
+      # a question mark if it is press the a dialog is shown which has the list.
+      #
+      # File types allowed by Apple are: pdf, doc, docx, rtf, pages, xls, xlsx, numbers
+      # zip, rar, plist, crash, jpg, png, mp4 or avi.
+      #
+      #
+      # @param review_attachment_path (String): The path to the attachment file.
+      def upload_review_attachment!(review_attachment_path)
+        raise 'cannot upload review attachment for live edition.' if self.is_live?
+
+        unless review_attachment_path && review_attachment_path.size > 0
```

Comment:
Small style issue:

Could you re-write this with an `if`. It is easier to reason about the conditional when its not negated.
As far as I understand, this is supposed to be `if !(A == nil || A.size > 0 )` ?